src/assets/js/react/components/FontManager/TemplateTooltip.js

```diff
@@ -0,0 +1,86 @@
+/* Dependencies */
+import React from 'react'
+import PropTypes from 'prop-types'
+
+/**
+ * @package     Gravity PDF
+ * @copyright   Copyright (c) 2020, Blue Liquid Designs
+ * @license     http://opensource.org/licenses/gpl-2.0.php GNU Public License
+ * @since       6.0
+ */
+
+export class TemplateTooltip extends React.Component {
+  /**
+   * PropTypes
+   *
+   * @since 6.0
+   */
+  static propTypes = {
+    id: PropTypes.string
+  }
+
+  /**
+   * Initialize component state
+   *
+   * @type {{ tooltip: boolean }}
+   *
+   * @since 6.0
+   */
+  state = {
+    tooltip: false
+  }
+
+  /**
+   * Toggle state for template usage information box
+   *
+   * @since 6.0
+   */
+  handleDisplayInfo = () => {
+    this.setState({ tooltip: !this.state.tooltip })
+  }
+
+  /**
+   * Handle auto highlighting of the information box content once clicked
+   *
+   * @param e: object
+   *
+   * @since 6.0
+   */
+  handleContentHighlight = e => {
+    e.target.focus()
+    e.target.select()
+
+    document.execCommand('copy')
+  }
+
+  /**
+   * Display template tooltip UI
+   *
+   * @since 6.0
+   */
+  render () {
+    const { id } = this.props
+    const { tooltip } = this.state
+    const textareaValue = `<style>${'\n'}.font-${id} {${'\n'}  font-family: ${id}, sans-serif;${'\n'}}${'\n'}</style>${'\n'}${'\n'}<div class="font-${id}">Text</div>`
```

Comment:
@debbieellis7 Let's go for readability over minimalism here. I recommend using Template literals to create the multiline string: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals